src/legacy/core_plugins/telemetry/server/telemetry_collection/get_kibana.ts

```diff
@@ -30,8 +48,17 @@ export function handleKibanaStats(server, response) {
 
   const { kibana, kibana_stats: kibanaStats, ...plugins } = response;
 
-  const platform = get(kibanaStats, 'os.platform', 'unknown');
-  const platformRelease = get(kibanaStats, 'os.platformRelease', 'unknown');
+  const os = {
+    platform: 'unknown',
+    platformRelease: 'unknown',
+    ...kibanaStats.os,
+  };
+  const formattedOsStats = Object.entries(os).reduce((acc, [key, value]) => {
+    return {
+      ...acc,
+      [`${key}s`]: [{ [key]: value, count: 1 }],
+    };
+  }, {});
```

Comment:
I'm not sure i understand the difference between the previous code and the current one?

Reply:
The previous code was only mapping `os.platforms[].platform` and `os.platformReleases[].platformRelease`. 

Now we are copying every element in `kibanaStats.os` (also including `distro` and `distroRelease`), but following the convention of `os[keyName + "s"] = [ { [keyName]: value, count: 1 } ]` (`os[KEY_IN_PLURAL] = Array<{ [KEY]: string, count: number }>`).

I can try changing it to something more strict and map the 2 new missing fields (`distro` and `distroRelease`), but that will result in the next time we add something to the `os` payload, we'll face this same issue again.